Cargo.toml

```
@@ -13,6 +13,10 @@ repository = "https://github.com/messense/cargo-options"
 [dependencies]
 anstyle = "1.0.2"
 clap = { version = "4.0.0", features = ["derive", "env", "wrap_help", "unstable-styles"] }
+serde = { version = "1", features = ["derive"], optional = true }
 
 [dev-dependencies]
 trycmd = { version = "0.14.0", features = ["examples"] }
+
+[features]
+serializable = ["serde"]
```

```suggestion
serde = ["dep:serde"]
```

I think it's fine to call this `serde`